uberspace/roles/uberspace_ephios/templates/ephios.ini.j2

```diff
@@ -18,19 +18,21 @@ gid = {{ ansible_user }}
 env = SECRET_KEY={{ django_secret }}
 env = DEBUG=False
 
-env = DATABASE_URL=mysql://{{ ansible_user }}:{{ db_password }}@localhost/{{ ansible_user }}_ephios
+env = DATABASE_URL={{ database_url }}
 
-env = ALLOWED_HOSTS={{ domain }}
+env = ALLOWED_HOSTS={{ domains|join(",") if use_main_documentroot else domain }}
 env = STATIC_URL=/static/
-env = STATIC_ROOT=/var/www/virtual/{{ ansible_user }}/{{ domain }}/static/
-env = MEDIA_ROOT=/var/www/virtual/{{ ansible_user }}/{{ domain }}/media/
+env = STATIC_ROOT=/var/www/virtual/{{ ansible_user }}/{{ "html" if use_main_documentroot else domain }}/static/
+env = STATIC_ROOT=/var/www/virtual/{{ ansible_user }}/{{ "html" if use_main_documentroot else domain }}/static/
 
+{% if not use_main_documentroot %}
 env = SITE_URL=https://{{ domain }}
+{% endif %}
 
-env = EMAIL_URL=smtp+tls://{{ smtp_mailbox }}@{{ domain }}:{{ smtp_password }}@{{ ansible_host }}:587
+env = EMAIL_URL=smtp+tls://{{ smtp_mailbox }}@{{ smtp_domain if use_main_documentroot else domain }}:{{ smtp_password }}@{{ ansible_host }}:587
 
-env = DEFAULT_FROM_EMAIL={{ smtp_mailbox }}@{{ domain }}
-env = SERVER_EMAIL={{ smtp_mailbox}}@{{ domain }}
+env = DEFAULT_FROM_EMAIL={{ smtp_mailbox }}@{{ smtp_domain if use_main_documentroot else domain }}
+env = SERVER_EMAIL={{ smtp_mailbox}}@{{ smtp_domain if use_main_documentroot else domain }}
 
 env = ADMINS="{{ admin_email }}"
 
```

Comment:
we also need a way to set our custom settings module here

Reply:
`DJANGO_SETTINGS_MODULE` should do the trick right?